### ISSUE TITLE
Can't login through bwlogin

### ISSUE THREAD

**Issue by songsanli** [blacs30/bitwarden-alfred-workflow#2]
Hi blacs30, thanks for your awesome job. I run into error when login and the error message is:

```
Your account could not be logged in for songsanli3@outlook.com Error Message: /bin/bash: /Users/i308587/Library/Application: No such file or directory
/bin/bash: /Users/i308587/Library/Application: No such file or directory

  error: option `--code <code>' argument missing
. 
```

I use https://bitwarden.com as server.

Do you have any idea about it?

**Comment by blacs30** [comment]
Hi @songsanli ,

thanks for opening this issue. I rewrote couple of scripts over the weekend and just released version 1.2.0. Can you please test that version. My hope is that it works now.

[version 1.2.0](https://github.com/blacs30/bitwarden-alfred-workflow/releases/tag/1.2.0)

**Comment by songsanli** [comment]
Thx, bug has been resolved!
